cmd/kpod/inspect.go

```diff
@@ -83,7 +83,10 @@ func inspectCmd(c *cli.Context) error {
 	}
 	server, err := libkpod.New(config)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "could not get container server")
+	}
+	if err = server.Update(); err != nil {
```

Comment:
I had tried adding it to libkpod.New(), but that seemed to break other things.  If I recall correctly, it had something to do with the fact that server.New() calls libkpod.New().